src/commands/add.ts

```diff
@@ -1,3 +1,5 @@
+import * as _ from 'lodash';
+import * as dashify from 'dashify';
```

Comment:
Can't we just use `lodash`'s `kebabCase` function since that's already installed? [link](https://lodash.com/docs/4.17.15#kebabCase)

Reply:
Why yes, yes we could

Reply:
Just a heads up, doing this instead makes the lodash import (much) smaller:

```
import kebabCase from 'lodash/kebabCase'
```